.github/workflows/main.yml

```diff
@@ -27,4 +27,3 @@ jobs:
     - run: npm ci
     - run: npm run build --if-present
     - run: npm test
-      
```

Comment:
This isn't really that important, but I suggest that you and Michael have the same vs.code settings when it comes to whitespace and newlines. Otherwise whenever someone makes a PR there will always be these unrelated changes.

Reply:
That's right. 
This is a blank mistake when I set up my initial CI.